timemachine/fe/atom_mapping.py

```diff
@@ -437,9 +417,8 @@ def leaf_filter_fxn(trial_core) -> bool:
         make_leaf_filter_fxn(),
     )
 
-    all_bond_cores = [_compute_bond_cores(mol_a, mol_b, marcs) for marcs in all_marcs]
```

Comment:
This is where the bulk of the performance gain comes from, and we never used the bonds.